src/quickhtml/tags.py

```diff
@@ -82,35 +92,36 @@ def render(self):
         Returns:
             str: The HTML representation of the tag and its child tags.
         """
-        # Apply tag attributes
-        if self.attrs:
-            ret_html = f"<{self.tag} "
-            for key, value in self.attrs.items():
-                # Translate the keyword argument class_ to class
-                if key == "class_":
-                    key = "class"
-
-                # Translate value None to 'none'
-                if value is None:
-                    value = "none"
-
-                # Replace underscores with hyphens
-                key = key.replace("_", "-")
-                ret_html += f"{key}='{value}' "
-            else:
-                ret_html = ret_html[:-1] + ">"
-        else:
-            ret_html = f"<{self.tag}>"
+
+        ret_html = f"<{self.tag} "
+
+        for key, value in self.attrs.items():
+            if key == "class_":
+                key = "class"
```

Comment:
Since you're messing with it anyways
```suggestion
            if key.endswith("_"):
                key = key[:-1]
```
It's something I've been meaning to do

Reply:
I just ended up using .rstrip(), should achieve the same though (just no if statement needed)